frontend/src/app/settings/modelsources/t4c-settings/t4c-instance-settings/t4c-instance-settings.component.html

```diff
@@ -3,108 +3,142 @@
-    <mat-card-title><h2>Manage T4C repositories</h2></mat-card-title>
+<div>
+  <div class="flex justify-between mr-[10px]">
+    <h2>Repositories</h2>
     <button
-      mat-icon-button
+      mat-stroked-button
       [disabled]="!instance"
       (click)="this.t4cRepoService.refreshRepositories(instance!.id)"
     >
       <mat-icon>sync</mat-icon>
     </button>
   </div>
-  <form [formGroup]="form" #formDirective="ngForm">
-    <mat-form-field appearance="fill">
-      <mat-label>Repository name</mat-label>
-      <input matInput formControlName="name" />
-      <mat-error *ngIf="form.controls.name.errors?.required"
-        >Please enter a name</mat-error
-      >
-      <mat-error *ngIf="form.controls.name.errors?.projectExistsError"
-        >Name already used</mat-error
-      >
-      <mat-error *ngIf="form.controls.name.errors?.pattern"
-        >The following characters are allowed: A-Z, a-z, 0-9, _, -
-      </mat-error>
-    </mat-form-field>
-    <div class="field-separator"></div>
-    <button
-      mat-flat-button
-      color="primary"
-      [disabled]="!form.valid"
-      (click)="createRepository()"
-    >
-      Add T4C repository
-    </button>
-  </form>
-  <mat-selection-list #repositoryList [multiple]="false" class="user-selection">
-    <mat-list-option
-      *ngFor="let repository of t4cRepoService.repositories | async"
-      [value]="repository"
-    >
-      <mat-icon mat-list-icon *ngIf="repository.status === 'LOADING'"
-        >sync</mat-icon
-      >
-      <mat-icon mat-list-icon *ngIf="repository.status === 'OFFLINE'"
-        >cloud_off</mat-icon
-      >
-      <mat-icon mat-list-icon *ngIf="repository.status === 'ONLINE'"
-        >cloud_queue</mat-icon
-      >
-      <mat-icon mat-list-icon *ngIf="repository.status === 'NOT_FOUND'"
-        >block</mat-icon
-      >
-      <mat-icon mat-list-icon *ngIf="repository.status === 'INITIAL'"
-        >cloud_upload</mat-icon
-      >
-      <mat-icon
-        mat-list-icon
-        *ngIf="repository.status === 'INSTANCE_UNREACHABLE'"
-        >sync_problem</mat-icon
-      >
-      <div>
-        <div mat-line>{{ repository.name }}</div>
-        <div mat-line>Repository ID: {{ repository.id }}</div>
-        <div mat-line>status: {{ repository.status }}</div>
+
+  <mat-card>
+    <form [formGroup]="form">
+      <div class="flex items-center">
+        <mat-form-field class="mb-[-1.25em]" appearance="fill">
+          <mat-label>Repository name</mat-label>
+          <input matInput formControlName="name" />
+          <mat-error *ngIf="form.controls.name.errors?.required"
+            >Please enter a name</mat-error
+          >
+          <mat-error *ngIf="form.controls.name.errors?.projectExistsError"
+            >Name already used</mat-error
+          >
+          <mat-error *ngIf="form.controls.name.errors?.pattern"
+            >The following characters are allowed: A-Z, a-z, 0-9, _, -
+          </mat-error>
```

Comment:
The margins don't look right: 
![image](https://github.com/DSD-DBS/capella-collab-manager/assets/23395732/ac9d3b41-7e85-4acc-9fe9-bf7b6fcd4135)